src/RMKERR.H

```diff
@@ -172,6 +172,13 @@ template< typename T> void MDbPrintf( 		// debug print a matrix
 		VDbPrintf( oMsk, NULL, pRow, nCol, fmt);
 	}
 }			// MDbPrintf
+// debugging aid: warn if limits invoked
+template< typename T> inline T bracketWarn( T vMin, T v, T vMax)
```

Comment:
What caused this guy to move to a different file?

Reply:
Yes, `CNGLOB` has [these includes](https://github.com/cse-sim/cse/blob/master/src/CNGLOB.H#L733-L738) almost at the end. The only thing is that in doing that, I will have to move some definitions around. An example is moving `typedef SI RC` before `#include rmkerr.h`.

Reply:
@nealkruis would it be ok to only move `#include rmkerr.h` while keeping the other set of `#include` at the bottom? Or should I move them all together?

Initially I was going to move them all together for organization, but it looks like it will take more effort than only moving `#include rmkerr.h` and I don't know if it is worth it.

Reply:
Maybe a question for @chipbarnaby. I'm not sure why those "more nested includes" are so low in the file or if there is a need to preserve the order of them.

Reply:
They depend on some definitions earlier in the files. At first glance, it looks like it is possible to reorganize all of it.